### DISCUSSION TITLE
Features comparison

### DISCUSSION THREAD

**Discussion by theom4** [cesanta/mongoose#2853] [Ask questions, get answers] [answered]
Just curious what are the key differences between LWIP and Mongoose?
I am using an STM32F7 board and I simply want to get a rough comparison

**Comment by scaprile** [comment, marked as answer]
This is not an issue. Please don't open an issue unless there is an actual issue.
Converted to discussion.
Please see our [documentation](https://mongoose.ws/documentation/),  and follow the guidelines in our [tutorials](https://mongoose.ws/documentation/#tutorials-and-examples).
Your question is to broad to be answered here, particularly because Mongoose can run over lwIP, or standalone, and the only thing they have in common is that both have a TCP/IP stack and can run in baremetal.

**Comment by theom4** [comment]
Sorry, my bad. Also thank you for your response. Sergey clarified it for me
